__Main__.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>Don't print these params manually, print them in code in case they change -_-</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-Farhad1353/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I understand you are 100% correct about this.</p><p>However, the only reason, I did print this is, I did not want you you to spend approximately 5 minutes each for the 1st stage of Random Forest , Ada Boost and Gradient Boost prior to executing the 2nd stage(final stage) of the program. </p>